maven-cache-clojure-plugin/pom.xml

```diff
@@ -56,6 +63,50 @@
                                             <outputs>
                                                 <cacheableBecause>compilation is CPU-bound</cacheableBecause>
                                             </outputs>
+                                            <inputs>
+                                                <properties>
+                                                    <property>
+                                                        <name>cleanAOTNamespaces</name>
+                                                    </property>
+                                                </properties>
+                                                <ignoredProperties>
+                                                    <ignore>temporaryOutputDirectory</ignore>
+                                                </ignoredProperties>
+                                            </inputs>
+                                        </execution>
+                                        <execution>
+                                            <id>test-clojure</id>
+                                            <outputs>
+                                                <cacheableBecause>test execution is time-consuming</cacheableBecause>
+                                            </outputs>
```

Comment:
❓No additional outputs like test reports?

Reply:
It uses `testOutputDirectory` which is declared below.